utils/statusCodes.ts

```diff
@@ -0,0 +1,12 @@
+// statusCodes.ts
+export const STATUS_CODES = {
+    UNPROCESSABLE_ENTITY: 422,
+    UNAUTHORIZED: 401,
+  };
+  
+  export const ERROR_MESSAGES = {
+    VALIDATION_ERROR: "Username and password field is required",
+    USER_NOT_FOUND: "Invalid username or password",
+    INVALID_CREDENTIALS: "Invalid username or password",
+  };
```

Comment:
Please, keep the error messages in `login.post.ts`. They aren't used anywhere else. Moving them outside of the "login.post.ts" makes it harder to follow the code.